plugins/destination/gremlin/client/client.go

```diff
@@ -0,0 +1,66 @@
+package client
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+
+	gremlingo "github.com/apache/tinkerpop/gremlin-go/v3/driver"
+	"github.com/cloudquery/plugin-sdk/plugins/destination"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"github.com/rs/zerolog"
+)
+
+type Client struct {
+	destination.UnimplementedUnmanagedWriter
+	destination.DefaultReverseTransformer
+	logger     zerolog.Logger
+	spec       specs.Destination
+	pluginSpec Spec
+	client     *gremlingo.DriverRemoteConnection
+}
+
+var AnonT = gremlingo.T__
+
+func New(ctx context.Context, logger zerolog.Logger, destSpec specs.Destination) (destination.Client, error) {
+	var err error
+	c := &Client{
+		logger: logger.With().Str("module", "gremlin").Logger(),
+		spec:   destSpec,
+	}
+	var spec Spec
+	if err := destSpec.UnmarshalSpec(&spec); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal gremlin spec: %w", err)
+	}
+	spec.SetDefaults()
+	if err := spec.Validate(); err != nil {
+		return nil, err
+	}
+
+	c.pluginSpec = spec
+
+	c.client, err = gremlingo.NewDriverRemoteConnection(spec.Endpoint+"/gremlin",
+		func(settings *gremlingo.DriverRemoteConnectionSettings) {
+			settings.TraversalSource = "g"
+			settings.LogVerbosity = gremlingo.Debug
```

Comment:
IMO it should be controlled by spec

Reply:
All gremlingo errors are logged as `TRACE` level in our logger (see logger.go) this is so that we don't miss any log messages and don't need to worry about converting between zerolog vs. gremlingo log levels.

Reply:
I mean that we should disable it by default and enable only if the user requests so

Reply:
Done